README.md

```diff
@@ -301,8 +302,8 @@ It is not possible to use different credentials for additional Backup Storage Lo
 
 ### Prerequisites
 
-* Velero 1.6.0 or later
-* AWS plugin must be installed, either at install time, or by running `velero plugin add velero/velero-plugin-for-aws:v1.2.1`
+* Velero 1.7.0 or later
```

Comment:
This shouldn't be updated, the addition bsl with their own credential is supported from v1.6.0

Reply:
Got it, thanks